db/db.sql

```diff
@@ -46,6 +46,25 @@ CREATE TABLE `job_errors` (
   FOREIGN KEY(jobID) REFERENCES jobs(jobID)
 );
 
+CREATE TABLE `labels` (
+  `id` integer not null primary key autoincrement,
+  `username` varchar(255) not null,
+  `value` varchar(255) not null,
+  `createdAt` datetime not null,
+  `updatedAt` datetime not null,
+  UNIQUE(username, value)
```

Comment:
Why don't we make the value itself be the unique key?

Reply:
What is the use case of username and value?

Reply:
If we did that then only one person could be associated with the label value. We want to be able to look up the labels for a person so we can populate an autocomplete, so we need to have both value and username here.

Reply:
When I log in as an admin, I think I should see all labels rather than just mine.

Reply:
We can get the labels for a user through the jobs table and the association with labels.